manager/assets/modext/widgets/resource/modx.tree.resource.js

```diff
@@ -401,6 +401,10 @@ Ext.extend(MODx.tree.Resource,MODx.tree.Tree,{
             if(menuindexField && o.result.object.menuindex !== undefined){
                 menuindexField.setValue(o.result.object.menuindex);
             }
+            var isfolderFieldCmb = Ext.getCmp('modx-resource-isfolder');
+            if(isfolderFieldCmb && o.result.object.isfolder !== undefined && typeof o.result.object.isfolder === 'boolean'){
```

Comment:
Is it possible to remove the `o.result.object.isfolder !== undefined` conditional? If I'm not mistaken the conditional `typeof o.result.object.isfolder === 'boolean'` returns false if `o.result.object.isfolder` is `undefined`. 

```suggestion
            if(isfolderFieldCmb && typeof o.result.object.isfolder === 'boolean'){
```